src/wazuh_modules/vulnerability_scanner/src/vulnerabilityScannerFacade.cpp

```diff
@@ -423,16 +423,22 @@ void VulnerabilityScannerFacade::start(
         clusterConfigurationChange(*stateDB);
 
         // Indexer connector initialization.
+        const auto& indexerConfig = policyManager.getIndexerConfiguration();
```

Comment:
reason to expand the scope ? in the line 440, we call with policyManager.getIndexerConfiguration() as parameter.

Reply:
You are right, the simplified constructor doesn't require this. I'll revert it.